### DISCUSSION TITLE
Override FS22 standard autobrake below 3 to 4 kph

### DISCUSSION THREAD

**Discussion by A3Ambition** [Mogli12/KeyboardSteer#781] [Ideas]
Hi Stefan,

First of all thanks for what you already did with your Mod to FS22 and a even bigger thanks for striving to even improve upon that state further.

I know that the feature was already present in an older release but with your latest additions (especially the reworked throttle control) it would change the game and the control of the vehicles if the ever present slow speed autobrake would be overwritten and cancelled out. The handbrake is working now so the MP issue should be covered I guess. 

Maybe you can add this back in?

Thanks a lot for your consideration.

**Comment by Mogli12** [comment]
Hi,

This is difficult. The auto brake is needed in some situations and you can only get rid of it by completely overwriting WheelsUtil:updateWheelsPhysics.

**Reply by A3Ambition** [reply]
Hi Stefan,

I did not know that. I thought it would be as simple as just lowering the threshold for when the Autobrake is kicking in. 

Sorry.

**Reply by A3Ambition** [reply]
Hi Stefan,

I saw that in Build 10 of the FS22 version of VCA you‘ve put in place the code to only have the lower brakeforce available above 1 m/s (3.6 Km/h). Would it be possible to potentially lower this to e.g. 1 km/h? That way the autobrake would remain active but you could control the vehicles much better at lower speeds?

Seeking guidance from you here.

Thanks so much.
